hack/common.sh

```diff
@@ -711,11 +711,11 @@ os::build::ldflags() {
 
   GO_VERSION=($(go version))
   if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.5') ]]; then
-    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitCommit" "${KUBE_GIT_COMMIT}")
-    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitVersion" "${KUBE_GIT_VERSION}")
+    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitCommit" "${OS_GIT_COMMIT}")
+    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitVersion" "${OS_GIT_VERSION}")
   else
-    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitCommit=${KUBE_GIT_COMMIT}")
-    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitVersion=${KUBE_GIT_VERSION}")
+    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitCommit=${OS_GIT_COMMIT}")
+    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitVersion=${OS_GIT_VERSION}")
```

Comment:
Users and clients want to know what version of Kube we are. We have to
keep it.  We want to get better at describing exactly what version we
include.  We can't really change /version since that is a Kube API